searx/plugins/hostnames.py

```diff
@@ -1,7 +1,21 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 # pylint: disable=too-many-branches
-"""In addition to rewriting/replace reslut URLs, the *hoostnames* plugin offers
-other features.
+""" .. note::
+
+    The **Hostname replace** plugin has been deprecated and replace by
+    **Hostnames plugin**.
```

Comment:
"deprecated" is the wrong word here:

    The **Hostname replace** plugin has been replace by the **Hostnames plugin**.

Reply:
Changed it.